fe/fe-core/src/main/java/org/apache/doris/load/loadv2/LoadJobScheduler.java

```diff
@@ -63,12 +64,18 @@ protected void runAfterCatalogReady() {
 
     private void process() throws InterruptedException {
         while (true) {
-            // take one load job from queue
-            LoadJob loadJob = needScheduleJobs.poll();
-            if (loadJob == null) {
+            if (!needScheduleJobs.isEmpty()) {
+                if (needScheduleJobs.peek() instanceof BrokerLoadJob && Catalog.getCurrentCatalog().getLoadingLoadTaskScheduler().isTaskQueueFull()) {
```

Comment:
Broker or Spark Load？

Reply:
Spark Load not use async loading task pool